history/characters/59000_dalaranian.txt

```diff
@@ -130,8 +131,18 @@
 		}
 		employer = 10014				# the Scourge
 	}
+	603.8.7={
+		add_trait=being_undead
+		add_trait=creature_lich
+		effect={
+			set_graphical_culture = lich
+			add_friend=60003 #Arthas
+		}
+
+	}
```

Comment:
Don't lose human trait?

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/76708761-78aa9300-670a-11ea-95a1-780df49451a4.png)

</details>

Reply:
Fixed in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/commit/ec1ba82dd8cec9e52abcee74d0b514dd8a2a714e.